src/components/Accordion/Accordion.style.ts

```diff
@@ -0,0 +1,120 @@
+import { rem } from 'polished';
+import styled, { css } from 'styled-components';
+
+import { ChevronDown, CircleWarning } from '../../icons';
+import { Paragraph } from '../../typography';
+
+import { grayscale } from '../../color';
+import { core } from '../../tokens';
+
+export const AccordionStyled = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: ${rem(8)};
+`;
+
+export const Wrapper = styled.div<{
+  format: 'basic' | 'secondary';
+  active: boolean;
+  disabled: boolean;
+}>`
+  ${({ disabled }) =>
+    disabled &&
+    css`
+      pointer-events: none;
+      opacity: 0.4;
+    `}
+  ${({ active, theme, format }) =>
+    active &&
+    css`
+      background-color: ${format === 'basic'
+        ? theme.name === 'dark'
+          ? grayscale(800)
+          : grayscale(50)
```

Comment:
This may be a good place to use a surface token instead?

Reply:
hmm sorry i'm looking at surface tokens https://vimeo.github.io/iris/sb/DSYS-8-toggle/?path=/story/tokens-color--surface and not seeing any that fit with the designs/correlate to these grays (`grayscale-800` and `grayscale-50`). am i missing something?